frontend/src/metabase/dashboard/components/DashboardGrid.jsx

```diff
@@ -175,14 +175,13 @@ class DashboardGrid extends Component {
     const desktop = dashboard.ordered_cards.map(this.getLayoutForDashCard);
     const mobile = generateMobileLayout({
       desktopLayout: desktop,
-      // We want to keep the heights for all visualizations equal not to break the visual rhythm
-      // Exceptions are text cards (can take too much vertical space)
-      // and scalar value cards (basically a number and some text on a big card)
```

Comment:
FWIW, removing the comment as it doesn't seem to carry enough value + it'd be a burden to modify it every time we need to add another height exception